src/app/models/indicator-query-opts.model.ts

```diff
@@ -12,5 +12,10 @@ export interface IndicatorQueryOpts {
         years?: string[];
         time_aggregation?: string;
         unit?: string;
+
+        // threshold indicator parameters
```

Comment:
TypeScript interfaces can be extended. These might be better added as a separate
```
interface ThresholdIndicatorQueryOpts extends IndicatorQueryOpts {
    threshold: Number;
    threshold_units: string;
    threshold_comparator: string;
}
```
Allows us to explicitly send the proper parameter type with a given indicator and better describe the type (e.g. for ThresholdIndicatorQueryOps, these three extra params are not optional.